docs/dictionary/command/call.lcdoc

```diff
@@ -21,46 +21,49 @@ Example:
 call "mouseUp" of button 1 of card 1
 
 Parameters:
-handler:
+handlerName:
 Any handler in the script of the specified object, along with
 any parameters that handler requires, separated by commas. The entire 
 handler including parameters must be enclosed in quotes.
 
-object:
-Any object reference. If no object is specified, LiveCode uses
+objectRef:
+Any <object reference>. If no object is specified, LiveCode uses
 the current object.
 
 Description:
 Use the <call> <command> to use a <handler> that's not in the normal 
 <message path>.
 
 The <call> <command> sends a <handler> <message> to the 
-<object(glossary)>. If the <script> of the <object(glossary)> doesn't 
+<object>. If the <script> of the <object> doesn't 
 <trap> the <handler> <message>, the <message> is <pass|passed> to the 
-next <object(glossary)> in the object's <message path>.
+next <object> in the <object|object's> <message path>.
 
```

Comment:
Unfortunately `<object(glossary)>` needs to stay like that because `<object>` is a valid html tag